2018-09-15-scipy-mm-nc/proposal/long.md

```diff
@@ -0,0 +1,60 @@
+The Long Description:
+
+This talk covers a tool developed for fast, interactive, browser-based
+visualization of datasets using Jupyter widgets developed for use with yt. 
+Jupyter widgets offer a unique opportunity to access the functionality in yt 
+using a graphical interface, 
+allowing for simultaneous control over the data and an 
+ease-of-use not immediately accessible through the API
+alone. Further, browser latency may be an issue with direct
+visualization of volumetric data with a widget. 
+We minimize latency that results from data transmission between yt and the
+browser by compiling to WebAssembly and pushing data to the browser. Operations
+on the data are subsequently done in the browser.
```

Comment:
I don't think it's necessary to duplicate the short abstract inside the long description. The reviewer will see both. Use the long description as an opportunity to elaborate on the short description.

Reply:
Ah ok! I wasn't sure if they'd see both or not! I'll restructure it.

Reply:
Ok I rewrote the beginning section of the long description to provide more clarity on the long description. I'm right at the word count limit for the short description, but about 100 words over for the long description. How strict are those limits?